api-samples/scripting/README.md

```diff
@@ -0,0 +1,24 @@
+# chrome.scripting API
+
+This sample demonstrates basic usage of the [chrome.scripting](https://developer.chrome.com/docs/extensions/reference/scripting/) API.
```

Comment:
suggestion rephrase to make it more clear what this example does (for people like me who don't know what the scripting API does...)
```suggestion
This sample demonstrates how to use the [chrome.scripting](https://developer.chrome.com/docs/extensions/reference/scripting/) API to inject JavaScript into web pages.
```